strawberry/asgi/__init__.py

```diff
@@ -194,12 +194,19 @@ async def handle_async_results(
                 {"data": None, "errors": [format_graphql_error(error)]},
                 operation_id,
             )
+        except asyncio.CancelledError:
```

Comment:
I think this should come before `except Exception as error:`?

here https://docs.python.org/3.8/library/asyncio-exceptions.html it says that it's using BaseException on 3.8 as base so it will be captured by `except Exception` above?